src/App.js

```diff
@@ -32,7 +48,10 @@ function App() {
               {token ? (
                 <Redirect to="/AddItem" />
               ) : (
-                <Home onSubmit={(e) => createToken(e)} />
+                <Home
+                  onSubmit={(e) => createToken(e)}
+                  onSharedToken={shareToken}
```

Comment:
`onSharedToken` doesn't make much sense for our use case. It sounds like a function that would share a token, maybe it would email the token to someone or something like that. We could improve the readability of our code by changing `onSubmit` to `createToken` and `onSharedToken` to `onTokenSubmit` or something like that.